src/components/templates/ChatPageTemplate.tsx

```diff
@@ -2,18 +2,27 @@ import * as S from './styled';
 import ChatNavbar from '../organisms/chat/chat-navbar/ChatNavbar';
 import ChatRoomList from '../organisms/chat/chat-list/ChatRoomList';
 import { useEffect, useState } from 'react';
-import axios from 'axios';
 import { useRecoilState } from 'recoil';
-import { MentorInfoAtom } from '@/recoil/atoms/MentorInfoAtom';
 import ChatSpace from '../organisms/chat/chat-space/ChatSpace';
-import CHAT from '@/apis/chatApi/chat';
+import CHAT from '@/apis/chat';
 import { ChatRoomListType, MentorInfoType } from '@/types/chat';
 import ChatMentorList from '../organisms/chat/chat-list/ChatMentorList';
 import { ChatRoomListAtom } from '@/recoil/atoms/ChatRoomListAtom';
+import USER from '@/apis/user';
 
 const ChatPageTemplate = () => {
   const [getChatRoomList, setGetChatRoomList] =
     useRecoilState<ChatRoomListType[]>(ChatRoomListAtom);
+  const [getMyId, setGetMyId] = useState();
```

Comment:
```suggestion
  const [getMyId, setMyId] = useState();
```
`setGetMyId` 보단 그냥 `set`으로 가는게 좋을 듯 싶습니다.

Reply:
컨벤션인줄 알았습니다 ㅋㅋㅋ 원래 변수에서 set만 붙이는건줄
변경했습니다.